disnake/ext/commands/params.py

```diff
@@ -929,6 +929,7 @@ def inject(function: Callable[..., Any]) -> Any:
 def option_enum(
     choices: Union[Dict[str, TChoice], List[TChoice]], **kwargs: TChoice
 ) -> Type[TChoice]:
+    """A utility function to create an enum type with the provided values."""
```

Comment:
Should have return type and other directives marked. Which version was this added in? 
![image](https://user-images.githubusercontent.com/71233171/163420897-0aadb3be-0284-4477-9b09-774285f6a74e.png)

Reply:
That method was added in 2.1 - updated the docs: https://github.com/DisnakeDev/disnake/pull/470/commits/1784630a21b3ed5fa17502bb7e25b84215562503